modules/efk-logging-fluentd-log-location.adoc

```diff
@@ -5,19 +5,30 @@
 [id='efk-logging-fluentd-log-location_{context}']
 = Configuring Fluentd log location
 
-Fluentd writes logs to a specified file or to the default location, `/var/log/fluentd/fluentd.log`, based on the `LOGGING_FILE_PATH` environment variable.
+Fluentd writes logs to a specified file or to the default location, `/var/log/fluentd/fluentd.log`, 
+based on the `LOGGING_FILE_PATH` environment variable.
+
+.Prerequisite
+
+Set cluster logging to the unmanaged state.
 
 .Procedure
 
 To set the output location for the Fluentd logs:
 
-. Edit the `LOGGING_FILE_PATH` parameter. You can specify a particular file or `STDOUT`:
+. Edit the `LOGGING_FILE_PATH`  parameter in the `fluentd` daemonset. You can specify a particular file or `STDOUT`:
```

Comment:
I think its or 'console'

Reply:
@jcantrill You mean like in the example right below it? Ugh...